sql/moz-fx-data-shared-prod/telemetry_derived/fx_health_ind_click_event_protection_v1/query.sql

```diff
@@ -0,0 +1,18 @@
+SELECT
+  submission_date,
+  COUNTIF(event_method = 'click' AND event_object = 'etp_toggle_off') AS disable_etp_cnt,
+  COUNTIF(event_method = 'click' AND event_object = 'etp_toggle_on') AS enable_etp_cnt,
+  COUNTIF(
+    event_method = 'click'
+    AND event_object = 'sitenotworking_link'
+  ) AS click_site_not_working,
+  COUNTIF(event_method = 'click' AND event_object = 'send_report_link') AS click_report_cnt,
+  COUNTIF(event_method = 'click' AND event_object = 'send_report_submit') AS submit_report_cnt,
+  COUNTIF(event_method = 'open') AS open_panel
+FROM
+  `moz-fx-data-shared-prod.telemetry.events`
```

Comment:
Would this pull from the events table that you built?

Reply:
No because that has the incorrect logic - I checked and I can't use the logic that was in that SQL, because what Romain is using here in this dashboard is different

Reply:
I can close this PR and add it to that table instead though, that's probably a better route